core/src/scss/utilities/variables/core/_core-color-maps.scss

```diff
@@ -15,18 +15,21 @@ $su-core-colors: (
   bg:                           $su-color-white,
   bg--reverse:                  $su-color-black,
 
-  text:                         $su-color-black,
-  text--high-contrast:          $su-color-black-true,
-  text--reverse:                $su-color-white,
+  card-border:                  #e3e3e3,
+
+  focus:                        $su-color-sky,
 
   link:                         $su-color-digital-blue,
-  link--hover:                  $su-color-black,
   link--alt:                    $su-color-digital-red,
   link--alt-hover:              $su-color-black,
+  link--hover:                  $su-color-black,
   link--reverse:                $su-color-white,
   link--reverse-hover:          $su-color-digital-red-light,
 
-  card-border:                  #e3e3e3,
+  text:                         $su-color-black,
+  text--high-contrast:          #000,
```

Comment:
If you've merged in Decanter 6.2.2., this would be `$su-color-black-true` instead of `#000`.